ses/index.cgi

```diff
@@ -42,12 +42,13 @@ sub main {
     elsif ( $message_type eq 'Notification' ) {
         my $notification = decode_json_wrapper( $message->{Message} ) // return;
 
-        my $notification_type = $notification->{notificationType} // '';
+        my $notification_type = $notification->{eventType} // $notification->{notificationType} // '';
```

Comment:
I'd rather check for the existence of the key here and error out if it is neither (with logging)